pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = [
```

Comment:
Is it better to include `Cython` too?

A new virtual environment requires manual installation of Cython

Reply:
no problem, added